docker/cdh-docker/conf/hadoop.conf

```diff
@@ -12,6 +12,6 @@ hibench.hadoop.examples.test.jar        /usr/lib/hadoop-mapreduce/hadoop-mapredu
 hibench.hdfs.master       hdfs://localhost:9000
 
 
-# Hadoop release provider. Supported value: apache, cdh5, hdp
+# Hadoop release provider. Supported value: apache
```

Comment:
Need to update the below line
`hibench.hadoop.release cdh5`

Reply:
I noticed this. Since it's in cdh-docker, I am not able to verify it. I think either leave it as-is or remove cdh-docker?

Reply:
This piece of code in docker may not run right now.

Reply:
ok. Let's merge this.